src/combinatorics/simple_graph/basic.lean

```diff
@@ -335,6 +335,8 @@ section finite
 
 variables [fintype V]
 
+instance fintype.locally_finite [decidable_rel G.adj] : locally_finite G := λ v, set_fintype _
```

Comment:
I'm pretty sure this instance is not necessary.

Reply:
Ah oops you might be right, I did something wrong and the instance wasn't registering

Reply:
Unless there's some complicated interaction somewhere I don't understand, this indicates that you don't need this instance:
```lean
instance fintype.locally_finite [decidable_rel G.adj] : locally_finite G :=
by apply_instance
```

Reply:
It would be worth extending the `locally_finite` doc string that `[decidable_rel G.adj]` and `[fintype V]` imply local finiteness (there's already a mention of this in "Implementation notes" in the module documentation).

Reply:
Nice catch!
bors r+